website/docs/components/time/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Time
+description: The Time component is used to display dates and time in a consistent format.
+caption: Displays dates and times in a consistent format.
+related: ['components/form/text-input']
+links:
+  figma: {Link}
```

Comment:
Nit: need to add link

Reply:
@andgen404

Reply:
Not sure if we have a link as there's no Figma component so we may want to remove

Reply:
Yep, I'd recommend removing this

Reply:
Removed, thanks